elixir/apps/domain/priv/repo/migrations/20250508192722_add_indexes_to_actor_group_memberships.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Domain.Repo.Migrations.AddIndexesToActorGroupMemberships do
+  use Ecto.Migration
+
+  @disable_ddl_transaction true
+
+  def change do
+    create(index("actor_group_memberships", [:group_id], concurrently: true))
+    create(index("actor_group_memberships", [:actor_id], concurrently: true))
```

Comment:
@bmanifold I just realized this one is redundant. The primary key index over `{actor_id, group_id}` covers the `actor_id` case. Open another PR to drop this one? Sorry!

Reply:
Happy to open another PR, but I'm not sure I'm following.  If that primary key index covers the `actor_id` case, why wouldn't it also cover the `group_id` case?

Reply:
It's a compound index. Compound indexes only work if your query matches the fields in the order the index was defined over. So queries using `actor_id` or `actor_id, group_id` would use the index, while queries for just `group_id` would not, since the b-tree index structure requires an actor_id to perform the first level lookup.

Reply:
Our AI overlords can explain it better ;-).

Reply:
Ah, ok, TIL!  I always thought a compound index had to use both fields during a query.

I'll open up a new PR